### PR TITLE
Added restore password to modal

### PR DESCRIPTION
**Pull request by Tivgres**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -23,4 +23,18 @@ $(function() {
...
+function showRestore() {
+    $(".login-form").hide();
+    $(".login-footer").hide();
+    $(".restore-form").show();
+    $(".restore-footer").show();
+}
+
+function showLogin() {
+    $(".login-form").show();
+    $(".login-footer").show();
```
Comment (drkmen) [inline app/assets/javascripts/application.js]: $(".login-form, .login-footer").show();

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -78,3 +78,11 @@ a, .link {
 .select2-container {
   width: 100% !important;
 }
+
+.restore-form {
+  display: none;
+}
```
Comment (drkmen) [inline app/assets/stylesheets/application.scss]: delete this and use `.d-none` class

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -78,3 +78,11 @@ a, .link {
...
+
+.restore-footer {
+  display: none;
+}
```
Comment (drkmen) [inline app/assets/stylesheets/application.scss]: same

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (Tivgres) [reply app/assets/stylesheets/application.scss]: NOPE! It does not work with bootstrap's display.

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (Tivgres) [reply app/assets/javascripts/application.js]: Okay :(

File: app/assets/stylesheets/application.scss (hunk 3 above)
Comment (Tivgres) [reply app/assets/stylesheets/application.scss]: Same doesn't work!

**Pull request merged**